internal/resources/grafana/resource_dashboard.go

```diff
@@ -362,7 +362,7 @@ func NormalizeDashboardConfigJSON(config interface{}) string {
 	// from the dashboard JSON other than "name" or "uid".
 	// Grafana will populate all other libraryPanel attributes, so delete them to avoid diff.
 	panels, hasPanels := dashboardJSON["panels"]
-	if hasPanels {
+	if hasPanels && panels != nil {
```

Comment:
The error message is:

```
panic: interface conversion: interface {} is nil, not []interface {}
```

Wouldn't perhaps the right check be?
```suggestion
	if _, ok := pabels.([]interface{}); ok && hasPanels {
```

Reply:
hasPanels is useless in that case. I think we only need the cast in fact: https://go.dev/play/p/qCqc3sQfkNk